x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/utils.ts

```diff
@@ -182,3 +182,20 @@ export const getPolicyTemplateInputOptions = (policyTemplate: CloudSecurityPolic
     icon: o.icon,
     disabled: o.disabled,
   }));
+
+export const getMaxPackageName = (
```

Comment:
I doesn't allow you to import it from https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/common/services/max_package_name.ts ?